lib/registries/addon/components/make-decision-dropdown/template.hbs

```diff
@@ -13,50 +13,54 @@
         </Button>
     </dd.trigger>
     <dd.content @class={{local-class 'Dropdown'}}>
-        <ReviewActionsList @registration={{@registration}} />
-        {{#each (get this.reviewsStateDecisionMap @registration.reviewsState) as |option|}}
-            <div>
-                {{#let (unique-id 'radio' option) as |uniqueId|}}
-                    <Input
-                        data-test-moderation-dropdown-decision-checkbox={{option}}
-                        @type='radio'
-                        @id={{uniqueId}}
-                        @value={{option}}
-                        checked={{if (eq this.decisionTrigger option) true false}}
-                        {{on 'click' (fn this.updateDecisionTrigger option)}}
-                    />
-                    <label
-                        for={{uniqueId}}
-                    >
-                        {{humanize option}}
-                    </label>
-                    <p
-                        data-test-moderation-dropdown-decision-description={{option}}
-                        local-class='Description'
-                    >
-                        {{get this.decisionDescriptionMap option}}
-                    </p>
-                {{/let}}
-            </div>
-        {{/each}}
-        <label for='moderator-comment'>
-            {{t 'registries.makeDecisionDropdown.additionalComment'}}
-        </label>
-        <Textarea
-            data-test-moderation-dropdown-comment
-            @id='moderator-comment'
-            @value={{this.comment}}
-            local-class='Comment'
-        />
-        <Button
-            data-test-moderation-dropdown-submit
-            @type='create'
-            disabled={{not this.decisionTrigger}}
-            local-class='SubmitButton'
-            {{on 'click' (perform this.submitDecision)}}
-            {{on 'click' dd.close}}
-        >
-            {{t 'registries.makeDecisionDropdown.submit'}}
-        </Button>
+        <ReviewActionsList @reviewActions={{this.reviewActions}} />
+        {{#if (eq @registration.reviewsState 'pending_withdraw_request')}}
+            {{t 'registries.makeDecisionDropdown.noActionAvailable'}}
+        {{else}}
+            {{#each (get this.reviewsStateDecisionMap @registration.reviewsState) as |option|}}
```

Comment:
Can the `noActionAvailable` translation go into the `{{else}}` block of the `{{each}}`? According to the state diagram, moderators shouldn't be able to do anything for `pendingWithdrawRequest` and `pendingEmbargoTermination` (and `initial` as well, but that's a slightly different case).

Reply:
I didn't realize the submit button would still be rendered if we did the each/else approach, so maybe broadening the condition to see if `(get this.reviewsStateDecisionMap @registration.reviewsState)` returns anything?